project/MicroService.scala

```diff
@@ -27,14 +29,15 @@ trait MicroService {
   lazy val compileScalastyle = taskKey[Unit]("compileScalastyle")
 
   lazy val microservice = Project(appName, file("."))
-    .enablePlugins(Seq(play.sbt.PlayScala,SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin) ++ plugins : _*)
+    .enablePlugins(Seq(play.sbt.PlayScala,SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory) ++ plugins : _*)
```

Comment:
What is ```SbtArtifactory``` ?

Reply:
it's a plugin needed for the new build system:
https://confluence.tools.tax.service.gov.uk/display/PLATOPS/Migrate+services+and+libraries+to+new+Build#MigrateservicesandlibrariestonewBuild-Updateyour%60project/plugins.sbt%60file